implementations/rust/ockam/ockam_api/src/nodes/service.rs

```diff
@@ -196,6 +193,12 @@ impl NodeManager {
         Ok(s)
     }
 
+    pub async fn configure_authorities(&mut self, ac: AuthoritiesConfig) -> Result<()> {
+        let v = Vault::default();
```

Comment:
Ideally we should use existing `Vault` instance. The idea is that Vault is pluggable, it's hard to achieve that if it's created many times in different places

Reply:
Changed in https://github.com/build-trust/ockam/pull/3336/commits/eb01611e5f7a513429a2f3df1665fc639aa25b62. Let me know if you want to have `configure_authorities` return an error instead of falling back to a default vault if none exists.

Reply:
Yeah, ideally it should return an error. Not critical for now though